src/Header.tsx

```diff
@@ -7,18 +7,34 @@ import searchIcon from "./assets/icons/searchIcon.svg";
 import logoPic from "./assets/images/logoPic.png";
 
 const Header = () => {
+  const [isMobile,setIsMobile] = useState(window.innerWidth <= 900);
```

Comment:
It's possiable that this isMobile will be used in more places in the code, so please move it to a general custom hook in the shared area.

Reply:
fixed